### PR TITLE
feat: user cart page

### PR DESCRIPTION
**Pull request by BelideNikhil**
added cart page to the e-comm

web view

image

mobile view 

image

### CODE REVIEW

File: pages/cart-page/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.cart-summary i {
+    margin-right: var(--margin-xs);
+}
+.cart-summary ~ .btn {
+    width: 100%;
+    margin: var(--margin-sm) 0;
+}
+.apply-coup-btn {
+    width: max-content;
+    margin: 0;
```
Comment (msdeshmukh009) [inline pages/cart-page/cart-page.css:58]: You are using `margin:0;` in multiple places maybe make a utility class for it.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

**Review by tanveertkd — APPROVED** [review]
This is good to go! Love the theme you picked. 👍

File: pages/cart-page/cart-page.css (hunk 1 above)
Comment (BelideNikhil) [reply pages/cart-page/cart-page.css:58]: I'm resetting the margins that I applied in the component library, once I'm done with ecomm, I'll update thra margins in CL and temove it here.

Thanks for the feedback

**Pull request merged**